### PR TITLE
Breaking removal by pk queries

### PR DESCRIPTION
**Pull request by joonatank**
## 🛠️ Changelog
- Removes byPk type GraphQL queries, replaced by relay queries.
- Fragments a lot of queries.
- Requires Backend PR https://github.com/City-of-Helsinki/tilavarauspalvelu-core/pull/1127
- First part of the reservation / reservation-unit API refactoring.

## 🧪 Test plan
-

## 🎫 Tickets
- [TILA-3238](https://helsinkisolutionoffice.atlassian.net/browse/TILA-3238)

[TILA-3238]: https://helsinkisolutionoffice.atlassian.net/browse/TILA-3238?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/admin-ui/src/component/reservations/requested/hooks/queries.ts [hunk 1]
```diff
@@ -70,6 +70,7 @@ const SPECIALISED_SINGLE_RESERVATION_FRAGMENT = gql`
   }
 `;
 
+// TODO replace with relay query
```
Comment (joonatank) [inline apps/admin-ui/src/component/reservations/requested/hooks/queries.ts:73]: Check that this still exists (in the API), i.e. the query works.

File: apps/admin-ui/src/spa/ReservationUnit/edit/queries.ts [hunk 2]
```diff
@@ -76,13 +73,6 @@ export const RESERVATIONUNIT_QUERY = gql`
         imageType
         imageUrl
       }
-      location {
-        addressStreetFi
-        addressZip
-        addressCityFi
-        longitude
-        latitude
-      }
```
Comment (joonatank) [inline apps/admin-ui/src/spa/ReservationUnit/edit/queries.ts:85]: Check that this was removed logically (not needed anymore).

**Review by vincit-matu — APPROVED** [review]
Looks okay 👍

File: apps/admin-ui/src/spa/ReservationUnit/edit/queries.ts (hunk 2 above)
Comment (joonatank) [reply apps/admin-ui/src/spa/ReservationUnit/edit/queries.ts:85]: Yeah. It's just done stupidly. There is a separate query on that page that fetches the unit and it's parameters with a pk. Could combine the queries into single one.

File: apps/admin-ui/src/component/reservations/requested/hooks/queries.ts (hunk 1 above)
Comment (joonatank) [reply apps/admin-ui/src/component/reservations/requested/hooks/queries.ts:73]: Fine. Still works. `reservationUnitByPk` got removed, the `reservationByPk` is removed in the next PR.

**Pull request merged**